### PR TITLE
fix: add more coins to onramper mapping

### PR DESCRIPTION
**Pull request by reallybeard**

### CODE REVIEW

File: packages/caip/src/adapters/onramper/index.ts [hunk 1]
```diff
@@ -10,28 +16,45 @@ export const AssetIdToOnRamperIdMap: Record<AssetId, OnRamperTokenId[]> = {
...
   [avalancheAssetId]: ['AVAX'],
+  [dogeAssetId]: ['DOGE'],
+  'eip155:1/erc20:0x111111111117dc0aa78b770fa6a738034120c302': ['1INCH'],
+  'eip155:1/erc20:0xbcca60bb61934080951369a648fb03df4f96263c': ['AUSDC'],
   'eip155:1/erc20:0x7fc66500c84a76ad7e9c93437bfc5ac33e2ddae9': ['AAVE_ECR20'],
   'eip155:1/erc20:0xbb0e17ef65f82ab018d8edd776e8dd940327b28b': ['AXS'],
   'eip155:1/erc20:0x4d224452801aced8b2f0aebe155379bb5d594381': ['APE'],
   'eip155:1/erc20:0x0d8775f648430679a709e98d2b0cb6250d2887ef': ['BAT_ERC20'],
   'eip155:1/erc20:0x4fabb145d64652a948d72533023f6e7a623c7c53': ['BUSD'],
+  'bip122:000000000000000000651ef99cb9fcbe/slip44:145': ['BCH'],
```
Comment (0xdef1cafe) [inline packages/caip/src/adapters/onramper/index.ts:27]: plz use imported asset id here

File: packages/caip/src/adapters/onramper/index.ts [hunk 2]
```diff
@@ -10,28 +16,45 @@ export const AssetIdToOnRamperIdMap: Record<AssetId, OnRamperTokenId[]> = {
...
   'eip155:1/erc20:0x3506424f91fd33084466f402d5d97f05f8e3b4af': ['CHZ'],
   'eip155:1/erc20:0xc00e94cb662c3520282e6f5717214004a7f26888': ['COMP'],
   'eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f': ['DAI', 'DAI_ERC20'],
   'eip155:1/erc20:0xf629cbd94d3791c9250152bd8dfbdf380e2a3b9c': ['ENJ'],
   'eip155:1/erc20:0x4e15361fd6b4bb609fa63c81a2be19d873717870': ['FTM'],
   'eip155:1/erc20:0xf57e7e7c23978c3caec3c3548e3d615c346e79ff': ['IMX'],
   'eip155:1/erc20:0x514910771af9ca656af840dff83e8264ecf986ca': ['LINK'],
+  'bip122:12a765e31ffd4059bada1e25190f6e98/slip44:2': ['LTC'],
```
Comment (0xdef1cafe) [inline packages/caip/src/adapters/onramper/index.ts:35]: and here

File: packages/caip/src/adapters/onramper/index.ts [hunk 3]
```diff
@@ -22,16 +35,28 @@ export const AssetIdToOnRamperIdMap: Record<AssetId, OnRamperTokenId[]> = {
...
+  'eip155:1/erc20:0x0000000000085d4780b73119b644ae5ecd22b376': ['TUSD'],
+  'eip155:1/erc20:0x04fa0d235c4abf4bcf4787af4cf447de572ef828': ['UMA'],
   'eip155:1/erc20:0x1f9840a85d5af5bf1d1762f925bdaddc4201f984': ['UNI', 'UNI-ERC20'],
   'eip155:1/erc20:0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48': ['USDC'],
   'eip155:1/erc20:0xdac17f958d2ee523a2206206994597c13d831ec7': ['USDT'],
   'eip155:1/erc20:0x2260fac5e5542a773aa44fbcfedf7c193bc2c599': ['WBTC'],
+  'eip155:1/erc20:0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2': ['WETH'],
+  'eip155:1/erc20:0x0bc529c00c6401aef6d220be8c6ea1667f6ad93e': ['YFI'],
+  'eip155:1/erc20:0xe41d2489571d322189246dafa5ebde1f4699f498': ['ZRX'],
+  'eip155:43114/erc20:0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e': ['USDC_CCHAIN'],
```
Comment (gomesalexandre) [inline packages/caip/src/adapters/onramper/index.ts:59]: Confirmed we good https://snowtrace.io/address/0xb97ef9ef8734c71904d8002f8b6bc66dd9c48a6e

**Review by gomesalexandre — APPROVED** [review]
Checked all the newly added `Assetid`s (keys) which are correct

**Pull request merged**